src/Contributors.js

```diff
@@ -20,4 +20,10 @@ export default [
   	facebook: "",
   	twitter: "aaryan7476",
   }
```

Comment:
@ayuushhh  As you can see all the checks have failed. As a hint, there is a small error at line 22. Try to find the error yourself this time.

Reply:
Sir i corrected the mistakes .Please check it.